stripe-attestation/src/main/java/com/stripe/attestation/StandardIntegrityManagerFactory.kt

```diff
@@ -0,0 +1,23 @@
+package com.stripe.attestation
+
+import android.content.Context
+import androidx.annotation.RestrictTo
+import com.google.android.play.core.integrity.IntegrityManagerFactory
+import com.google.android.play.core.integrity.StandardIntegrityManager
+
+/**
+ * Factory for creating a [StandardIntegrityManager].
+ *
+ * This is used to allow for easier testing.
+ */
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+fun interface StandardIntegrityManagerFactory {
+    fun create(): StandardIntegrityManager
+}
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+class RealStandardIntegrityManagerFactory(private val appContext: Context) : StandardIntegrityManagerFactory {
```

Comment:
Should we make this take in `Application` instead to make memory leaks impossible?

Reply:
https://github.com/stripe/stripe-android/pull/9464/commits/417c4b8ab88e435f41fec6f3467218e77a5d0e68